tests/test_config.ts

```diff
@@ -1,21 +1,25 @@
-import  {
-    ApolloClient,
-    InMemoryCache,
-    HttpLink,
-} from "@apollo/client/core";
-import fetch from "cross-fetch";
+import fetch, { Headers } from "cross-fetch";
+require('isomorphic-fetch');
 
 // Declare URLs for algorithm 1, algorithm 2, and backend
 export const BACKEND_URL = process.env.BACKEND_URL || "http://localhost:4000";
```

Comment:
Do these env vars come from the calling workflow? ie if we have our own e2e file with a different URL, is that were it's sourced from?

Reply:
Exactly 👍